doc/source/ray-air/examples/huggingface_text_classification.ipynb

```diff
@@ -544,150 +473,75 @@
       "outputs": [
         {
           "data": {
-            "application/vnd.jupyter.widget-view+json": {
-              "model_id": "28ff97b9821d495088a0711191c3e12e",
-              "version_major": 2,
-              "version_minor": 0
-            },
-            "text/plain": [
-              "  0%|          | 0/9 [00:00<?, ?ba/s]"
-            ]
-          },
-          "metadata": {},
-          "output_type": "display_data"
-        },
-        {
-          "data": {
-            "application/vnd.jupyter.widget-view+json": {
-              "model_id": "ed9f698c7c4f46ff9c520ed0597b6bf6",
-              "version_major": 2,
-              "version_minor": 0
-            },
-            "text/plain": [
-              "  0%|          | 0/2 [00:00<?, ?ba/s]"
-            ]
-          },
-          "metadata": {},
-          "output_type": "display_data"
-        },
-        {
-          "data": {
-            "application/vnd.jupyter.widget-view+json": {
-              "model_id": "4f73054b701f4684b3a44793d10d4a0f",
-              "version_major": 2,
-              "version_minor": 0
-            },
             "text/plain": [
-              "  0%|          | 0/2 [00:00<?, ?ba/s]"
+              "{'train': Dataset(num_blocks=1, num_rows=8551, schema={sentence: string, label: int64, idx: int32}),\n",
+              " 'validation': Dataset(num_blocks=1, num_rows=1043, schema={sentence: string, label: int64, idx: int32}),\n",
+              " 'test': Dataset(num_blocks=1, num_rows=1063, schema={sentence: string, label: int64, idx: int32})}"
             ]
           },
+          "execution_count": 11,
           "metadata": {},
-          "output_type": "display_data"
+          "output_type": "execute_result"
         }
       ],
       "source": [
-        "encoded_datasets = datasets.map(preprocess_function, batched=True, fn_kwargs=dict(tokenizer=tokenizer))"
+        "import ray.data\n",
+        "\n",
+        "ray_datasets = ray.data.from_huggingface(datasets)\n",
+        "ray_datasets"
       ]
     },
     {
       "cell_type": "markdown",
       "metadata": {
-        "id": "256fOuzjhYbY"
+        "id": "2C0hcmp9IrJQ"
       },
       "source": [
-        "For Ray AIR, instead of using 🤗 Dataset objects directly, we will convert them to [Ray Datasets](https://docs.ray.io/en/latest/data/dataset.html). Both are backed by Arrow tables, so the conversion is straightforward. We will use the built-in `ray.data.from_huggingface` function."
+        "We can them write the function that will preprocess our samples. We just feed them to the `tokenizer` with the argument `truncation=True`. This will ensure that an input longer that what the model selected can handle will be truncated to the maximum length accepted by the model.\n",
+        "\n",
+        "We use a `BatchMapper` to create a Ray AIR preprocessor that will map function to the dataset in a distributed fashion. It will be lazily ran during training and prediction."
```

Comment:
```suggestion
        "We use a `BatchMapper` to create a Ray AIR preprocessor that will map the function to the dataset in a distributed fashion. It will be lazily ran during training and prediction."
```